### DISCUSSION TITLE
Turn target mod's metronome into an togglable option

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#20006] [Ideas] [answered]
This is how I use target

https://user-images.githubusercontent.com/40531871/187096873-1106c185-d9ff-4b59-aa43-53a4484a5e0f.mp4

I use Target to fix my offset perfectly ( max 3x retries of couple secs and Im accurate within ~0.6ms )
But the metronome of target mode really messes me up, its hard to focus on the actual beatmap's rhythm

Is it possible to make target mod's metronome a togglable option?

**Comment by peppy** [comment, marked as answer]
I think making it a toggle is fine.

**Reply by peppy** [reply]
#20012
